lib/aws_auditor/aws.rb

```diff
@@ -9,8 +9,37 @@ class AwsConfig < Hash
 
   class AWSSDK
     def self.authenticate(environment)
-      aws = Aws::SharedCredentials.new(profile_name: environment)
-      Aws.config.update({region: 'us-east-1', credentials: aws})
+      shared_credentials = Aws::SharedCredentials.new(profile_name: environment)
+      Aws.config.update({region: 'us-east-1', credentials: shared_credentials})
+
+      iam = Aws::IAM::Client.new
+      iam.list_mfa_devices.mfa_devices.each do |mfadevice|
+        mfa_serial_number = mfadevice.serial_number
+        if mfa_serial_number
```

Comment:
Lets take this block out of the iterator since only one device is supported.

From:  http://aws.amazon.com/iam/faqs/#MFA_FAQs
```
Q. Can I have multiple authentication devices active for my AWS account?
Yes. Each IAM user can have its own authentication device. However, each identity (IAM user or root account) can be associated with only one authentication device.
```